src/styles/header/header-variables.scss

```diff
@@ -0,0 +1,5 @@
+$HTMLfont-family: roboto, sans-serif;
+$HTMLfont-weight: 500;
+$HTMLfont-size: 12px;
+$HTMLcolor: black;
+$HTMLbutton-background: #00acdc;
```

Comment:
```suggestion
$HTMLfont-weight: 500;
$HTMLfont-size: 12px;
$HTMLcolor: black;
$HTMLbutton-background: $color-blue;
```

Reply:
Should I assign variable "$color-blue" from another block ?